### ISSUE TITLE
False positive with methods used for first arg to setTimeout/setInterval

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-linter#651]
See details + example at https://github.com/mozilla/addons-server/issues/2560

``` javascript
window.setInterval(tabmanager.updateClosedCount, settings.get('badgeCounterInterval'));
```

Causes:

`setTimeout or setInterval must have function as 1st arg`

**Comment by EnTeQuAk** [comment]
This will be fixed once https://github.com/mozilla/addons-linter/issues/1197 lands and replaces the current implementation. Linking these two to have the test ready for later.
